web/src/bootstrap_typeahead.ts

```diff
@@ -304,7 +305,7 @@ export class Typeahead<ItemType extends string | object> {
         this.updateElementContent = options.updateElementContent ?? true;
         this.hideAfterSelect = options.hideAfterSelect ?? (() => true);
         this.hideOnEmptyAfterBackspace = options.hideOnEmptyAfterBackspace ?? false;
-
+        this.getCustomItemClassname = options.getCustomItemClassname ?? (() => "");
```

Comment:
This should be `this.getCustomItemClassname = options.getCustomItemClassname` to keep it undefined if it's not specified